okta/resource_okta_policy_mfa.go

```diff
@@ -192,9 +199,16 @@ func syncFactor(d *schema.ResourceData, k string, f *sdk.PolicyFactor) {
 func syncAuthenticator(d *schema.ResourceData, k string, authenticators []*sdk.PolicyAuthenticator) {
 	for _, authenticator := range authenticators {
 		if authenticator.Key == k {
-			_ = d.Set(k, map[string]interface{}{
-				"enroll": authenticator.Enroll.Self,
-			})
+			if authenticator.Constraints != nil {
+				_ = d.Set(k, map[string]interface{}{
+					"enroll":      authenticator.Enroll.Self,
+					"constraints": strings.Join(authenticator.Constraints.AaguidGroups, ","),
```

Comment:
@monde authenticator.Constraints.AaguidGroups' order is not maintained, so there will be constant update